extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -359,12 +381,16 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
                 }
 
                 let global = getGlobal();
+                let isPolyfill = (fetch as any).polyfill;
                 if (_config.disableFetchTracking === false) {
                     _hooks.push(InstrumentFunc(global, strFetch, {
                         // Add request hook
                         req: (callDetails: IInstrumentCallDetails, input, init) => {
                             let fetchData: ajaxRecord;
-                            if (_fetchInitialized && !_isDisabledRequest(null, input, init)) {
+                            if (_fetchInitialized && 
+                                    !_isDisabledRequest(null, input, init) && 
+                                    // If we have a polyfil and XHR instrumented then let XHR report otherwise we get duplicates
+                                    !(isPolyfill && _xhrInitialized)) {
```

Comment:
This is now required, as previously the XHR requests where actually throwing an internal exception (and not logging) because the responseType was "Blob".

With the responseType fix and not this check is was actually reporting the request for both the fetch and underlying (polyfilled) XHR request.